examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,404 @@
80489 - 0.3531363820423749im, -0.35788786882845036 + 0.18057836964629678im, 0.17133550110477333 + 0.3624036703730536im, 0.36668081281797327 - 0.16197980043769386im, -0.1525174287942926 - 0.37071647947274017im, -0.37450801266953376 + 0.14295461757145544im, 0.13329766431001788 + 0.3780529155133954im, 0.3813488535265341 - 0.12355292854757727im, -0.11372682763048422 - 0.3843936561856749im, -0.3871853183514709 + 0.10382583248765614im, 0.09385646336926674 + 0.3897220015889509im, 0.39200203537823564 - 0.08382528555278071im, -0.07373890501951168 - 0.3940239182146283im, -0.3957863185974429 + 0.06360396410419812im, 0.05342713712081654 + 0.3972880759068388im, 0.3985282011681576 - 0.04321512596715629im, -0.03297465571140883 - 0.39950587770319324im, -0.4002204616680222 + 0.022712470163323374im, 0.012435327433167463 + 0.4006714824769971im, 0.40085864311265085 - 0.0021499954811794023im, 0.008136752339431992 - 0.40078182032129583im, -0.40044106469419166 - 0.0184181417430448im, -0.02868740197279545 + 0.39983660063422904im, 0.39896882620815033 + 0.03893777025942064im, 0.04916249627487312 - 0.3978383128844025im, -0.3964458051568056 - 0.05935484657765365im, -0.06950810904716966 + 0.39479222005426035im, 0.39287864653685556 + 0.07961559730387789im, 0.0896706551126502 - 0.3907063447787261im, -0.3882767453381822 - 0.09966666076614164im, -0.10959703144551036 + 0.3855914482156166im, 0.38265222179983466 + 0.11945522755549393im, 0.12923475703102658 - 0.3794610017034902im, -0.37601988948839776 - 0.13892917961255596im, -0.14853211108724307 + 0.3723311512815596im, 0.3683972162828193 + 0.15803722749325363im, 0.16743826928437075 - 0.3642206751651247im, -0.35980427836845563 - 0.17672904545218254im, -0.18590343760316458 + 0.3551509342885233im, 0.3502637073614882 + 0.1949554039878608im, 0.2038789834797125 - 0.34514581604584604im, -0.33980063070275657 - 0.21266829950055172im, -0.22131756388828625 + 0.334231671373094im, 0.32844260540973946 + 0.2298210806733758im, 0.23817324937737172 - 0.322437244464191im, -0.31621953596962044 - 0.24636856388878825im, -0.25440157363414634 + 0.3097935105631298im, 0.3031629362091899 + 0.26226657768051775im, 0.2699557734642881 - 0.2963293815893444im, -0.28928334681256634 - 0.2774501849359395im, -0.2846838052583661 + 0.28197161235123785im, 0.2742027760003744 + 0.29143115373043477im, 0.29703625929900734 - 0.26544137592122335im, -0.2545034504217247 - 0.29994466070413445im, -0.29724536413228575 + 0.23936984904848227im, 0.21757055542769113 + 0.2848392342067677im, 0.2589345565262642 - 0.1874616980884428im, -0.14991645959452302 - 0.21867585550726582im, -0.16816745093201246 + 0.10905400958561935im, 0.07090556142073211 + 0.11574173981531792im, 0.07030068156155729 - 0.04062419079799972im, -0.02028272687861378 - 0.037275927064832806im, -0.01711242427516849 + 0.008749823851945im, 0.003240148839174281 + 0.0067594032382922im, 0.0022865609036269953 - 0.0010247711432102616im, -0.00027570938946322465 - 0.0006600691074608816im, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0};\n",
```

Comment:
This output is a little overwhelming, is there something important in this output that should be focused on? can we look at just that instead? maybe we can use the `PulseSequence` instead of always going to OpenQASM

Reply:
The printout is indeed too long. Switch to a different gate that has shorter definition. PulseSequence doesn't have an interface to visualize its body beside going for qasm script.